### ISSUE TITLE
Remove russian language file

### ISSUE THREAD

**Issue by Zodiac1978** [pluginkollektiv/antispam-bee#189]
Russian language has its own language pack!
https://translate.wordpress.org/locale/ru/default/wp-plugins/antispam-bee

So we can remove the Russian language files from the plugin. 😃

**Comment by Zodiac1978** [comment]
After this is done we can close #15 finally.

**Comment by Zodiac1978** [comment]
@krafit Why not 2.7.2? This is a low hanging fruit and an easy fix.

**Comment by krafit** [comment]
@Zodiac1978 yep it is. But with #180 the next release will be more than a 2.7.X 🙃
